workshops/book-dash/2023-Nov-BookDash-10.md

```diff
@@ -0,0 +1,320 @@
+   * The Book Dash Planning Committee used the rubrics (as explained in the online chapter) to score the applications during the review. They met online to discuss applications, frame feedback and conclude their selection process.
+   * Relevant sub-chapter: [https://the-turing-way.netlify.app/community-handbook/bookdash/bookdash-application](https://the-turing-way.netlify.app/community-handbook/bookdash/bookdash-application)
+   * In the future, applications will include a wider range of timeslots for participation, and may also benefit from additional explanations of the type of work which is completed during a Book Dash. 
+### Hosting sessions 
+
+   * Along with Anne, we started a new issue to create the Book Dash Planning Committee subchapter ([https://github.com/the-turing-way/the-turing-way/issues/3367)](https://github.com/the-turing-way/the-turing-way/issues/3367)). The main idea was to create checklists for day lead(s) and session host(s). 
+   * We circulated these WIP checklists through the planning committee so that they can be tested live during the book dash itself and we also received some feedback on it.
+   * The checklists involved tasks for both the primary and secondary day leads and session hosts.
+   * For the day leads, the checklist was divided into 3 parts: before the day they lead, during the day they lead, and after the day they lead ends.
+   * For the session hosts, the checklist was divided into 3 parts (for both contribution and social sessions): before session, during session, and after the session. The checklist provided a technical orientation to hosting as well as describing the role of host.
+
+
+### Accessibility Assessment 
+
+   * The Book Dash organizing committee sought input from the community on ways to improve the accessibility of the event for people from marginalized groups.
+   * Two social events were held in Spanish.
+   * Collaborative documents were moved to the Framapad platform as a more screen-reader-accessible option than HackMD. Framapad was also an improvement over the Software Sustainability Institute's Etherpad instance, which was unstable with multiple simultaneous participants.
+   * The Accessibility Working Group was joined by two colleagues from Metadocencia, Patricia Lotto and  
+   * Jesica Formoso, who shared expertise in leading accessible events and educational experiences
+   * The Accessibility Working Group met early in the Book Dash to plan work on the TTW accessibility Policy and review the chapter "Planning Inclusive and Accessible Events" for the Guide to Collaboration authored by Sophia Batchelor.
+   * Patricia and Jesica provided complete reviews of the chapter. Liz provided advice and started discussions at the Book Dash and on GitHub on the overall structure and also considerations of the identities of readers and people who will benefit from this work. 
+### Book Dash Planning Committee (all of us should add our info and bio)
+
+[ Image will be placed here ]
```

Comment:
Do we have a photo of all of us, actually?

Reply:
Sadly, no